$NetBSD: patch-ac,v 1.1.1.1 1999/02/04 00:06:38 hubertf Exp $

I got this patch from Adam Sampson <azz@josstix.demon.co.uk>, the
author of xhippo.


--- xhippo.c.orig	Sat Jan  9 21:29:39 1999
+++ xhippo.c	Wed Feb  3 01:47:12 1999
@@ -391,7 +391,11 @@
 
   /* Create the filelist widget. */
   filelist = gtk_list_new();
+#if 0
   gtk_container_add(GTK_CONTAINER(scroll), filelist);
+#else
+  gtk_scrolled_window_add_with_viewport(GTK_SCROLLED_WINDOW(scroll), filelist);
+#endif
   gtk_signal_connect(GTK_OBJECT(filelist), "selection_changed",
 		     GTK_SIGNAL_FUNC(handle_listclick), NULL);
   gtk_widget_set_name(filelist, "filelist");
